de.metas.business/src/main/java/de/metas/pricing/conditions/service/impl/PricingConditionsRepository.java

```diff
@@ -119,9 +119,9 @@ public PricingConditions getPricingConditionsById(@NonNull final PricingConditio
 	@VisibleForTesting
 	PricingConditions retrievePricingConditionsById(@NonNull final PricingConditionsId id)
 	{
-		final int discountSchemaRecordId = id.getDiscountSchemaId();
+		final int discountSchemaRecordId = id.getRepoId();
 
-		final PricingConditionsId pricingConditionsId = PricingConditionsId.ofDiscountSchemaId(discountSchemaRecordId);
+		final PricingConditionsId pricingConditionsId = PricingConditionsId.ofRepoId(discountSchemaRecordId);
```

Comment:
this is pointless...

Reply:
It's not. I had to change to `ofRepoId` from `ofDiscountSchemaId` because otherwise a repoIdAware test wa failing.